be/src/http/action/stream_load.cpp

```diff
@@ -238,6 +238,10 @@ Status StreamLoadAction::_on_header(HttpRequest* http_req, StreamLoadContext* ct
 
     TNetworkAddress master_addr = _exec_env->master_info()->network_address;
 
+    if (!http_req->header(HTTP_TIMEOUT).empty()) {
+        ctx->timeout_second = std::stoi(http_req->header(HTTP_TIMEOUT)); 
```

Comment:
It will thrown the std::invalid_argument  exception which could not be catch in the out side.

Reply:
I will add a try catch block